app/api/contact/specs/contact.spec.js

```diff
@@ -27,7 +27,7 @@ describe('contact', () => {
         })
         .then(() => {
           expect(mailer.send).toHaveBeenCalledWith({
-            from: '"Uwazi" <no-reply@uwazi.io',
+            from: '"Uwazi" <no-reply@uwazi.io>',
```

Comment:
Not sure if this is the time to address this, but the fact that we send all our mails as "Uwazi" is kind of confusing.  People use a document management platform "powered by Uwazi", but a lot of users would expect the mail to come from "Plan International", or "CEJIL", not from "Uwazi", as 90% of the people using the platform (other than admins) have no idea what Uwazi is.

Could we just change this to the collection name?  But some collection names are whole books, so not sure.  Perhaps a configurable option (another?) in collection settings?  Should we do this here and now? (especially since this opens up the scenario to sending of a LOT more emails).